question1.py

```diff
@@ -0,0 +1,15 @@
+# Question1 
+
+choice = input("A. Length of list\nB. Display first 3 numbers\nC. Display sum of odd numbers\nD. Number of duplicate numbers\nE. Display list without duplicate numbers\nEnter Your choice: ")
+numbers  = [2,4,5,2,12,44,5,1,2,3]
+if choice.lower()=="a":
+    print("Length of list: ",len(numbers))
+elif choice.lower()=="b":
+    print("First 3 numbers: ",numbers[:3])
+elif choice.lower()=="c":
+    print("Sum of odd numbers",sum([x for x in numbers if x%2!=0]))
+elif choice.lower()=="d":
+    ducplicates_numbers = {x :numbers.count(x) for x in numbers if numbers.count(x) >=2 }
```

Comment:
I think this will have a performance issues.

Let's say I have a list where 2 repeats 1 lakh times, 3 repeats 10K times, 4 repeats 2 lakh times then at that time, even the count of 2 exists in the dictionary like `{2: 1 lakh}`, when it comes to second 2, it will again calculates this and overrides the `{2: 1 lakh}` with `{2: 1 lakh}` with same value.

Please make this more robust and optimize the code. 

Before starting any new question, please try to think about all the scenarios that comes in your mind and find a robust solution of the problem.

We are here to find how we can complete the problem efficiently, not to just make it work by doing coding. Let me know if you have any questions and queries. Thanks.